executor_manager/services/sandbox/manager.py

```
@@ -0,0 +1,961 @@
+            sandbox: Sandbox to run in
+            execution: Execution to run
+            timeout: Timeout in seconds
+        """
+
+        def on_running(exec: Execution):
+            self._repository.save_execution(exec)
+
+        def on_error(exec: Execution):
+            self._repository.save_execution(exec)
+
+        def on_complete(exec: Execution):
```

Ruff (0.14.10)

Unused function argument: `exec`

(ARG001)